lib/reports/worldwide_organisation_translations_report.rb

```diff
@@ -0,0 +1,95 @@
+module Reports
+  class WorldwideOrganisationTranslationsReport
+    CSV_HEADERS = [
+      "Item",
+      "Count",
+      "---",
+      "Worldwide organisation",
+      "Translation",
+      "Office",
+      "Office has translation",
+      "Contact has translation",
+      "Corporate information page",
+      "Corporate information page has translation",
+      "Worldwide organisation has translation",
+    ].freeze
+
+    def report
+      path = Rails.root.join("tmp/worldwide-organisation-translations_#{Time.zone.now.strftime('%d-%m-%Y_%H-%M')}.csv")
+
+      CSV.open(path, "wb", headers: CSV_HEADERS, write_headers: true) do |csv|
+        worldwide_organisations_count(csv)
+        worldwide_offices_count(csv)
+        worldwide_corporate_information_pages_count(csv)
+
+        organisations = WorldwideOrganisation.all.includes(:translations, offices: [contact: [:translations]])
+
+        organisations.each do |organisation|
+          organisation.translations.each do |translation|
+            organisation.offices.each do |office|
+              csv << skip_rows(3) + [
+                organisation.name,
+                translation.locale,
+                office.title,
+                human_bool(translation.locale == :en),
+                human_bool(office.contact.translations.map(&:locale).include?(translation.locale)),
+              ]
+            end
+
+            organisation.corporate_information_pages.where(state: :published).find_each do |page|
+              csv << skip_rows(3) + [organisation.name, translation.locale] + skip_rows(3) + [
+                page.slug,
+                human_bool(page.translations.map(&:locale).include?(translation.locale)),
+              ] + skip_rows(1)
+
+              cip_only_translations = page.translations.map(&:locale) - organisation.translations.map(&:locale)
+
+              next if translation.locale != :en || cip_only_translations.empty?
```

Comment:
This is slightly awkward, it's the CIPs which are translated into a language which the worldwide org is not. Skipping if locale is not :en so that only 1 row gets created per org.